application/deed/searchdeed/views.py

```diff
@@ -168,6 +187,11 @@ def show_internal_server_error_page():
 
 
 @searchdeed.errorhandler(status.HTTP_503_SERVICE_UNAVAILABLE)
+def service_unavailable_error(e):
+    return redirect(url_for('searchdeed.show_internal_server_error_page'))
```

Comment:
yeah so from a BFE point of view it doesn't matter whether its a temporary service unavailability (503) or if its an internal server error (500), we just need to redirect the user to an appropriate error page. That being said, we were previously raising a 503 but i think the actual code returned might be a 500 which explains why the acceptance test environment was returning an unhandled 500 error on Nigel and mandy's machines. So i have changed this now to a 500 to see if this would solve that problem.